content/sensu-core/0.29/guides/securing-rabbitmq-guide.md

```diff
@@ -0,0 +1,162 @@
+---

```

Comment:
I’d recommend changing the filename to `guides/securing-rabbitmq.md`

Reply:
:heavy_check_mark: